homework-02/modules/Importer.js

```diff
@@ -0,0 +1,30 @@
+import * as fs from 'fs'
+import * as syspath from 'path'
+import * as csvjson from 'csvjson'
+
+class Importer {
+
+  constructor () {
+
+  }
+
+  import (path) {
+    return new Promise((resolve) => {
+      resolve (this.importSync(path))
+    })
+  }
+
+  importSync (path) {
+    const fileList = fs.readdirSync(path)
```

Comment:
it seems like `eslint` doesn't work for your IDE:), so, please, add `eslint` to your npm precommit command